src/healthData/HealthDataUtils.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2015 Adobe Systems Incorporated. All rights reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, maxerr: 50 */
+/*global define, $ */
+
+define(function (require, exports, module) {
+    "use strict";
+    
+    var ExtensionManager = require("extensibility/ExtensionManager");
+    
+    /**
+     * Get the OS Version
+     * @return {String} version of OS 
+     */
+    function getOSVersion() {
+        var winOSStrings = {
+            "5.0" : "Windows 2000",
+            "5.1" : "Windows XP",
+            "5.2" : "Windows XP | Windows Server 2003 | Windows Home Server",
```

Comment:
I don't think we should catch it just yet -- I'm pretty sure there are still some using Brackets on XP, and it's way more useful to have the actual OS name on the server rather than just an User Agent string.
Also, it won't hurt to leave XP in for now.